modules/iphone/com.omorandi/1.1.2/example/app.js

```diff
@@ -0,0 +1,74 @@
+var showDialog = function() {
```

Comment:
This looks like example code from 3rd party software. Is it necessary to have this file?